packages/components-js/tests/a11y/specs/others/carousel.a11y.ts

```diff
@@ -146,12 +154,29 @@ test('should expose correct initial accessibility tree', async ({ page }) => {
   expect(await getAttribute(slide3, 'aria-label')).toBe('3 of 3');
 });
 
-test.fixme('should expose correct initial accessibility tree when aria prop is defined', async ({ page }) => {
-  await initCarousel(page, { aria: "{'aria-label': 'Other Heading'}" });
+test('should expose correct aria-label on splide element if slotted heading is set', async ({ page }) => {
+  await initCarousel(page, { heading: false, slottedHeading: true });
   const splide = await getSplide(page);
 
-  // await expectA11yToMatchSnapshot(page, splide, { message: 'splide', interestingOnly: false });
+  expect(await getAttribute(splide, 'aria-labelledby')).toBe('heading');
+});
+
+test('should expose correct aria-label when aria prop is defined and heading is set', async ({ page }) => {
+  await initCarousel(page, { aria: "{'aria-label': 'Other heading'}" });
```

Comment:
Maybe extract the `Other heading` into a const and use in the `expect` below. Also goes for the other test.